test/monitor_test.go

```diff
@@ -30,6 +33,32 @@ func runMonitorServer() *server.Server {
 	return RunServer(&opts)
 }
 
+// Runs a clustered pair of monitor servers for testing the /routez endpoint
+func runMonitorServerClusteredPair() (*server.Server, *server.Server) {
+	resetPreviousHTTPConnections()
+	opts := DefaultTestOptions
+	opts.Port = CLIENT_PORT
+	opts.HTTPPort = MONITOR_PORT
+	opts.HTTPHost = "localhost"
+	opts.Cluster.Host = "127.0.0.1"
+	opts.Cluster.Port = 10223
+	opts.Routes = server.RoutesFromStr("nats-route://127.0.0.1:10222")
+
+	s1 := RunServer(&opts)
+
+	opts2 := DefaultTestOptions
+	opts2.Port = CLIENT_PORT + 1
+	opts2.HTTPPort = MONITOR_PORT + 1
+	opts2.HTTPHost = "localhost"
+	opts2.Cluster.Host = "127.0.0.1"
+	opts2.Cluster.Port = 10222
+	opts2.Routes = server.RoutesFromStr("nats-route://127.0.0.1:10223")
+
+	s2 := RunServer(&opts2)
+
```

Comment:
You may want to call `checkClusterFormed(t, s1, s2)` here to ensure cluster is formed, instead of using `time.Sleep(2 * time.Second)` in the caller. You would need to pass `t *testing.T` as parameter to `runMonitorServerClusteredPair()`.